### PR TITLE
mon,osd: various octopus feature bits

### PR DESCRIPTION
**Pull request by liewegas**

### CODE REVIEW

File: src/test/cli/monmaptool/feature-set-unset-list.t [hunk 1]
```diff
@@ -11,64 +11,64 @@
...
   MONMAP FEATURES:
       persistent: [kraken(1)]
       optional:   [none]
       required:   [kraken(1)]
   
   AVAILABLE FEATURES:
-      supported:  [kraken(1),luminous(2),mimic(4),osdmap-prune(8),nautilus(16)]
-      persistent: [kraken(1),luminous(2),mimic(4),osdmap-prune(8),nautilus(16)]
+      supported:  [kraken(1),luminous(2),mimic(4),osdmap-prune(8),nautilus(16),octopus(32)]
+      persistent: [kraken(1),luminous(2),mimic(4),osdmap-prune(8),nautilus(16),octopus(32)]
```
Comment (tchaikov) [inline src/test/cli/monmaptool/feature-set-unset-list.t]: might want to update `TEST_mon_features()` in `qa/standalone/mon/misc.sh`. 

for instance.
1. check that we have "octopus" in monitor features
1. `.monmap.features.persistent | length == *6*`

Comment (liewegas) [reply src/test/cli/monmaptool/feature-set-unset-list.t]: fixed!

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2019-04-03_02:18:56-rados-wip-sage2-testing-2019-04-02-1625-distro-basic-smithi/ pass

the mimic upgrade failure is a problem with mimic, see https://github.com/ceph/ceph/pull/27351

File: src/include/ceph_features.h [hunk 2]
```diff
@@ -121,11 +122,13 @@ DEFINE_CEPH_FEATURE_RETIRED(22, 1, BACKFILL_RESERVATION, JEWEL, LUMINOUS)
 
 DEFINE_CEPH_FEATURE(23, 1, MSG_AUTH)         // 3.19 req (unless nocephx_require_signatures)
 DEFINE_CEPH_FEATURE_RETIRED(24, 1, RECOVERY_RESERVATION, JEWEL, LUNINOUS)
```
Comment (neha-ojha) [inline src/include/ceph_features.h]: we should probably fix this at some point, LUNINOUS -> LUMINOUS

**Pull request merged**